src/copick_utils/segmentation/segmentation_from_picks.py

```diff
@@ -128,26 +76,41 @@ def segmentation_from_picks(radius, painting_segmentation_name, run, voxel_spaci
         seg = segs[0]
 
     segmentation_group = zarr.open_group(seg.path, mode="a")
-    
-    # Ensure that the tomogram Zarr is available
-    tomogram_group = zarr.open(tomogram.zarr(), "r")
-    multiscale_levels = list(tomogram_group.array_keys())
-
-    # Iterate through multiscale levels to paint picks as spheres
-    for level in multiscale_levels:
-        level_data = tomogram_group[level]
-        shape = level_data.shape
-
-        # Load or initialize the segmentation array in memory for the current level
-        if level in segmentation_group:
-            painting_seg_array = np.array(segmentation_group[level])
-        else:
-            painting_seg_array = np.zeros(shape, dtype=np.uint16)
+    highest_res_name = "0"
+
+    # Get the highest resolution dimensions and create a new array if necessary
+    highest_res_shape = segmentation_group[highest_res_name].shape if highest_res_name in segmentation_group else tomogram.shape
+    if highest_res_name not in segmentation_group:
+        segmentation_group.create(highest_res_name, shape=highest_res_shape, dtype=np.uint16, overwrite=True)
+
+    # Initialize or load the highest resolution array
+    highest_res_seg = segmentation_group[highest_res_name][:]
+    highest_res_seg.fill(0)
+
+    # Paint picks into the highest resolution array
+    highest_res_seg = from_picks(pick_set, highest_res_seg, radius, pickable_object.label, voxel_spacing)
 
-        scale_factor = tomogram_group.attrs.get('multiscales', [{}])[0].get('datasets', [{}])[int(level)].get('scale', 1)
-        scaled_radius = int(radius / scale_factor)        
+    # Write back the highest resolution data
+    segmentation_group[highest_res_name][:] = highest_res_seg
 
-        # Once all picks are painted at this level, write the array to the Zarr store
-        segmentation_group[level] = from_picks(pick_set, painting_seg_array, scaled_radius, pickable_object.label, voxel_spacing)
+    tomogram_zarr = zarr.open(tomogram.zarr(), "r")
+
+    # Downsample to create lower resolution scales
+    multiscale_metadata = tomogram_zarr.attrs.get('multiscales', [{}])[0].get('datasets', [])
+    for level_index, level_metadata in enumerate(multiscale_metadata):
+        if level_index == 0:
+            continue
+
+        level_name = level_metadata.get("path", str(level_index))
+        expected_shape = tuple(segmentation_group[level_name].shape)
```

Comment:
This should likely be 

```python 
expected_shape = tuple(tomogram_zarr[level_name].shape)
```

As we want to match the expected shape of the tomogram.